### ISSUE TITLE
含BN的PB模型转换失败。

### ISSUE THREAD

**Issue by a-paw-fish** [Tencent/TNN#832]
Converte Tensorflow's Op arcface/embd_extractor/resnet_v2_50/block4/unit_3/block_v2/conv1/BatchNorm/cond/FusedBatchNorm , type = FusedBatchNorm, failed, may be some node is not const。
/home/zy/zy_work/MNN/tools/converter/source/common/writeFb.cpp:687: These Op Not Support:  Tensorflow::FusedBatchNorm | 
在转换PB为mnn模型时失败，怀疑是BN层可能含有is_training的占位符，这可能是导致该失败的原因？目前我只有PB模型（没有训练代码），怎样修改才能模型转换成功呢？

**Comment by gttiankai** [comment]
你这个是 MNN 模型转换失败的bug，我们这边是 TNN。
